ext/date/php_date.c

```diff
@@ -2263,17 +2263,19 @@ PHPAPI zval *php_date_instantiate(zend_class_entry *pce, zval *object) /* {{{ */
 
 /* Helper function used to store the latest found warnings and errors while
  * parsing, from either strtotime or parse_from_format. */
-static void update_errors_warnings(timelib_error_container *last_errors) /* {{{ */
+static void update_errors_warnings(timelib_error_container **last_errors) /* {{{ */
 {
 	if (DATEG(last_errors)) {
 		timelib_error_container_dtor(DATEG(last_errors));
 		DATEG(last_errors) = NULL;
 	}
-	if (last_errors->warning_count || last_errors->error_count) {
-		DATEG(last_errors) = last_errors;
-	} else {
-		timelib_error_container_dtor(last_errors);
+
+	if ((*last_errors)->warning_count || (*last_errors)->error_count) {
```

Comment:
Need to check that `last_errors != NULL`.

Reply:
It can't be, but I will add a check.